src/mode/modeHandler.ts

```diff
@@ -505,6 +505,7 @@ export class ModeHandler implements vscode.Disposable {
 
     async handleKeyEvent(key: string): Promise<Boolean> {
         if (key === "<c-r>") { key = "ctrl+r"; } // TODO - temporary hack for tests only!
+        if (key === "<ctrl-[>") { key = "<esc>"; } // TODO - temporary hack because package.json maps ctrl+[ to the escape command
```

Comment:
Correct. `<ctrl-[>` is only in test code.

I would _like_ to have a test to verify that pressing `ctrl+[` results in the right action, but perhaps a comment in the test to reference package.json is adequate. What do you think?

Something like this:
```
test("can be activated", async () => {
        let activationKeys = ['<esc>'];
        // ctrl+[ is mapped to the vim_esc command by package.json
...
    });
```